app/gui2/src/components/GraphEditor/NodeWidget.vue

```diff
@@ -43,16 +40,17 @@ const sameInputParentWidgets = computed(() =>
 )
 const nesting = computed(() => (parentUsageInfo?.nesting ?? 0) + (props.nest === true ? 1 : 0))
 
-const selectedWidget = computed(() => {
-  return registry.select(
+const selectedWidget = shallowRef<WidgetModule<WidgetInput> | undefined>()
+const updateSelection = withCtx(() => {
+  selectedWidget.value = registry.select(
     {
       input: props.input,
       nesting: nesting.value,
     },
     sameInputParentWidgets.value,
   )
-})
-
+}, getCurrentInstance())
+watchEffect(() => updateSelection())
```

Comment:
Why is it better? I cannot find docs of withCtx in google.

Reply:
It was a necessary change after vue update. Otherwise any context usage (i.e. injects) within `registry.select` fails, because it is executed outside of component context. This change here just made it behave as it it used to in this specific instance. I couldn't really find a good way to avoid using those injects in the first place.